packages/babel-runtime/helpers/esm/iterableToArray.js

```diff
@@ -1,3 +1,11 @@
 export default function _iterableToArray(iter) {
-  if (Symbol.iterator in Object(iter) || Object.prototype.toString.call(iter) === "[object Arguments]") return Array.from(iter);
-}
\ No newline at end of file
+  if (
+    Array.isArray(iter)
+    || typeof iter === 'string'
+    || (typeof Symbol === 'function' && Symbol.iterator in Object(iter))
+    || (iter && 'length' in iter)
+    || (typeof Map !== 'undefined' && iter instanceof Map)
+    || (typeof Set !== 'undefined' && iter instanceof Set)
```

Comment:
es6-shim provides both Map and Set but doesn’t provide Symbol.iterator when native Symbols aren’t supported; so these lines are needed.